support-models/src/main/scala/com/gu/support/workers/states/SendThankYouEmailState.scala

```diff
@@ -63,8 +64,7 @@ object SendThankYouEmailState {
     user: User,
     sfContactId: SfContactId,
     product: DigitalPack,
```

Comment:
`product: DigitalPack,` also has a billing period in it, but it's wrong as we didn't actually know when we populated it.  I will look to address this as part of https://trello.com/c/Mwb7aMda/3379-add-discriminator-to-producttype